### ISSUE TITLE
Vet Center phone numbers in service accordions

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#26688]
## Issue Description
Phone numbers entered into plain text fields on Drupal should be made accessible when rendered on the front end. This behavior can be observed in the Suicide prevention accordion on Nashville's main page in preview (nodeID- 3877). 

**The duplicate content also observed in this accordion is content related and will not be addressed as part of this issue.** 

## Current behavior

<details>
<summary> As seen on Nashville Vet Center </summary>

![image](https://user-images.githubusercontent.com/55411834/123686798-6b38ef80-d80d-11eb-92c8-3feab3ab7288.png)

</details>

## Expected behavior

<details>
<summary> Screenshot with example expected behavior</summary>

![image](https://user-images.githubusercontent.com/55411834/123686928-8e639f00-d80d-11eb-8abc-9139cc07d786.png)

</details>

## Dev notes
- Test to see what happens if the number has already been formatted appropriately - can we apply this filter to all phone numbers of restrict it to  **National content** only 
- local content is rich text and editors may have wrapped it during entry

---
## Tasks
- [ ] Phone number handling

## Acceptance Criteria
- [ ] Phone numbers are accessible when rendered on the front end

**Comment by mmiddaugh** [comment]
Looks great! Thank you!
